soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-alibaba-dubbo/pom.xml

```diff
@@ -38,6 +38,16 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>dubbo</artifactId>
+            <version>${alibaba.dubbo.version}</version>
```

Comment:
why add this?

Reply:
AlibabaDubboServiceBeanPostProcessor.java will use <com.alibaba>. Should I change the scope to test?

Reply:
of course